sky/backends/cloud_vm_ray_backend.py

```diff
@@ -3175,9 +3175,17 @@ def _run_setup(setup_cmd: str) -> int:
             returncode = _run_setup(f'{create_script_code} && {setup_cmd}',)
             if returncode == 255:
                 is_message_too_long = False
-                with open(setup_log_path, 'r', encoding='utf-8') as f:
-                    if 'too long' in f.read():
-                        is_message_too_long = True
+                try:
+                    with open(os.path.expanduser(setup_log_path),
+                              'r',
+                              encoding='utf-8') as f:
+                        if 'too long' in f.read():
+                            is_message_too_long = True
+                except Exception:  # pylint: disable=broad-except
```

Comment:
Should we except certain error type?

Reply:
This detection for messages too long is not critical, and we want to make it more conservative here by catching any exceptions. Previously, we got an error for FilesNotFound and the setup crashed.

Reply:
Added a comment there : )